sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/util/ModelHelper.java

```diff
@@ -196,4 +204,36 @@ public static ShareFileItemProperties transformFileProperty(FileProperty propert
         return new InternalShareFileItemProperties(property.getCreationTime(), property.getLastAccessTime(),
             property.getLastWriteTime(), property.getChangeTime(), property.getLastModified(), property.getEtag());
     }
+
+    public static HandleItem transformHandleItem(com.azure.storage.file.share.implementation.models.HandleItem handleItem) {
```

Comment:
Any reason for the fully qualified name here?

Reply:
There are two different types named `HandleItem` here, one in implementation and one in the public API